packages/Meshfree/test/tstNearestNeighborOperator.cpp

```diff
@@ -112,20 +107,8 @@ TEUCHOS_UNIT_TEST_TEMPLATE_1_DECL( NearestNeighborOperator, unique_source_point,
     DataTransferKit::NearestNeighborOperator<DeviceType> nnop(
         comm, source_points, target_points );
 
-    Kokkos::View<double *, DeviceType> source_values( "in" );
-    Kokkos::View<double *, DeviceType> target_values( "out" );
-
-    // violate pre condition of apply (source not properly sized)
```

Comment:
... and this

Reply:
Same reason as before: we don't throw a DTK exception anymore and I don't really care about it. The user just needs to be careful.